Backend/lib/payment/src/Lib/Payment/Domain/Action.hs

```diff
@@ -53,13 +54,58 @@ createOrderService ::
   (Payment.CreateOrderReq -> m Payment.CreateOrderResp) ->
   m Payment.CreateOrderResp
 createOrderService merchantId personId orderId createOrderReq createOrderCall = do
-  existingOrder <- runInReplica $ QOrder.findById orderId
-  whenJust existingOrder $ \_ -> throwError (InvalidRequest "Payment order already exists")
-  createOrderResp <- createOrderCall createOrderReq -- api call
-  paymentOrder <- buildPaymentOrder merchantId personId orderId createOrderReq createOrderResp
-  Esq.runTransaction $
-    QOrder.create paymentOrder
-  pure createOrderResp
+  mbExistingOrder <- runInReplica $ QOrder.findById orderId
+  case mbExistingOrder of
+    Nothing -> do
+      createOrderResp <- createOrderCall createOrderReq -- api call
+      paymentOrder <- buildPaymentOrder merchantId personId orderId createOrderReq createOrderResp
+      Esq.runTransaction $
+        QOrder.create paymentOrder
+      pure createOrderResp
+    Just existingOrder -> do
+      sdk_payload <- buildSDKPayload createOrderReq existingOrder
+      pure
+        Payment.CreateOrderResp
+          { status = existingOrder.status,
+            id = "ordeh_xxxxxxxxxxxxxxxxxxxx", -- not saved in db
```

Comment:
Can make these fields optional in shared-kernel instead of hardcoding, if we don't use them at frontend @hkmangla

Reply:
I mean these three , not saved in db

```
id = "ordeh_xxxxxxxxxxxxxxxxxxxx", -- not saved in db
.......
    Juspay.SDKPayload
      { requestId = "12398b5571d74c3388a74004bc24370c", -- not saved in db
        service = "in.juspay.hyperpay", -- not saved in db
        payload
      }
```